org.eclipse.scout.rt.server.commons.test/src/test/java/org/eclipse/scout/rt/server/commons/servlet/HttpProxyTest.java

```diff
@@ -253,4 +307,169 @@ protected void assertRewriteResponseHeaders(Map<String, String> expectedHeaders,
     verify(res, atLeast(0)).setHeader(anyString(), anyString());
     verifyNoMoreInteractions(res);
   }
+
+  @Test
+  public void testProxyRequest_noContent() throws IOException {
+    testProxyRequestWithStatusCodeAndContent_Internal(200, new byte[]{}, true);
+    testProxyRequestWithStatusCodeAndContent_Internal(200, new byte[]{}, false);
+  }
+
+  @Test
+  public void testProxyRequest_200() throws IOException {
+    testProxyRequestWithStatusCodeAndContent_Internal(200, new byte[]{0x01}, true);
+  }
+
+  @Test
+  public void testProxyRequest_numerousRequests() throws IOException {
+    // keep number of requests low on CI (at least as long as we do not run tests in parallel), however can be increased for testing locally
+    testProxyRequestWithStatusCodeAndContent_Internal(200, new byte[]{0x01}, true, 25);
+  }
+
+  @Test
+  public void testProxyRequest_404() throws IOException {
+    testProxyRequestWithStatusCodeAndContent_Internal(404, new byte[]{0x02}, true);
+  }
+
+  @Test
+  public void testProxyRequest_500() throws IOException {
+    testProxyRequestWithStatusCodeAndContent_Internal(500, new byte[]{0x02}, true);
+  }
+
+  @Test
+  public void testProxyRequest_largeContent() throws IOException {
+    // keep size pseudo-large (could be larger) on CI (at least as long as we do not run tests in parallel), however can be increased for testing locally
+    byte[] content = new byte[65536 * 4096];
```

Comment:
Maybe still to much for CI? Test seems to fail on CI with OOME; either decrease size here or ignore it (only use it for local testing).

Reply:
Suggest to decrease size.

Reply:
Fixed in https://github.com/eclipse-scout/scout.rt/compare/f48c418ea822284b166f9970e4124a60e383bd63..e779671a1c3dd3fdd03df96227b29f95f566207b